Sensors.cpp

```diff
@@ -1,8 +1,22 @@
 #include "Sensors.hpp"
+#define SEALEVELPRESSURE_HPA (1013.25)
+#define DHTTYPE DHT22         // DHT 22 (AM2302)
+#define DHTPIN 15
 
 // Humidity sensor
 Adafruit_AM2320 am2320 = Adafruit_AM2320();
 
+//BME280
+ Adafruit_BME280 bme;           // BME280 I2C
+//AHT10
+ Adafruit_AHTX0 aht;
+ Adafruit_Sensor *aht_humidity, *aht_temp;
+//SHT31
+ Adafruit_SHT31 sht31 = Adafruit_SHT31();
+//DHT22
+ DHT dht(DHTPIN, DHTTYPE);
```

Comment:
Please here only declare the variables, on each init method of each library please instance or initialize the variables, in special for DHT dht(x,x) because it need pass this values from the own parameters on the own init method.

Reply:
I made many changes but I don't understand the construction of a library. "H&T_type" is like "device_type" for choose the H&T model of sensor but don't know how to choose the sensor? automatically? test every model by i2c and then the DTH22 and put the H&T_type value?